cfgov/unprocessed/apps/form-explainer/yarn.lock

```diff
@@ -1,4 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I suspect this file will pop up again, as these autogenerated yarn.lock files came up for me through some process of flushing the yarn cache and re-installing everything.

Reply:
I was assuming that nothing would appear, given that there is no package.json file anymore. But I could be wrong. Are you saying that _on this branch_, the `yarn.lock` files reappeared for you?

Reply:
Yes. If you throw out all the cache files and lock files and re-`yarn install`, we not only get things in the cache that should be updated (which seems like we should do periodically), but we also get these auto-generated empty lock files:

<img width="1138" alt="Screen Shot 2020-08-25 at 2 13 00 PM" src="https://user-images.githubusercontent.com/704760/91211861-6da65680-e6dd-11ea-9269-3faae58071fd.png">

Reply:
Strange. I don't understand why Yarn would do anything in that folder in the absence of a `package.json` file. Maybe this is added incentive for us to try out Yarn 2, which I believe overhauled the lock system.

Reply:
Added ab34f192c8bb99979ec17d2d2370fdc9df73400b to prevent us from trying to run `yarn` when `package.json` exists but has no deps listed.